classes/03class/exercises/c03-docker06/gepifanio/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.12.0
+ENV NAME=Guilherme
```

Comment:
Please have a look at this again. The expectation is to pass the name during the docker run, as an argument. The question on README.md is `When the image runs, it should output the following text: Hello `name`, where the value of name is passed during the container execution`

Reply:
that's it. Thanks Jay